catroid/src/main/java/org/catrobat/catroid/content/XmlHeader.java

```diff
@@ -64,12 +64,26 @@
 	private double platformVersion = 0;
 	@SuppressWarnings("unused")
 	private String programLicense = "";
+
+	//---------------------------------------------------------------------------
+	//              *** Catrobat Remix Specification ***
+	//---------------------------------------------------------------------------
+	//  Keep in mind that the remixOf-field is used and supposed to be modified
+	//  by Catroweb's web application *only*!
+	//  Once new Catrobat programs get uploaded, Catroweb automatically
+	//  updates the remixOf-field and sets the program as being remixed.
+	//  In order to do so, Catroweb takes the value from the remixUrlsString-field and
+	//  assigns it to the remixOf-field.
+	//
+	//  With that said, the only correct way to set a new remix-URL (e.g. when
+	//  two programs get merged locally) is to assign it to the remixUrlsString-field.
+	//---------------------------------------------------------------------------
```

Comment:
^^ I liked your comment and in this case it's definitely necessary. It isn't obvious how the field is handled at all.

My question is still _why?_
Why you reuse the url to temporary save the remix info? Why you set the remixOf information on the server?